doc.html

```diff
@@ -8,225 +8,75 @@
   <link type="text/css" rel="stylesheet" href="dist/doc.css" />
```

Comment:
You may include the bootstrap css easily to make the style compatible with the main system.
Take a look at the templates: http://getbootstrap.com/getting-started/#examples-framework